src/control/settings/Settings.h

```diff
@@ -887,4 +888,8 @@ class Settings {
      * e.g. "en_US"
      */
     std::string preferredLocale;
+
+public:
+    Palette* palette;
```

Comment:
**Question:** Who is the owner of the Palette, Control or Settings or both? Or are both Palette* distinct?

Reply:
Oh, the one in control was only a leftover, from the initial implementation attempt. There should be only one palette and it's in settings.

Reply:
Ok then it should be a std::unique_ptr

Reply:
:heavy_check_mark: